.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```

Comment:
Could you please exclude .idea/ entirely because IDE stuff should not be part of the repo.